tox.ini

```diff
@@ -137,13 +133,13 @@ application-import-names = deploy
 
 description = run Mypy (static type checker)
 
-basepython = python3.8
-
-skip_install = True
```

Comment:
Don't skip install of `hyperlink` and its dependencies in `mypy` environment, because that helps `mypy` file type hints in the dependencies.